From ce51032e49c460d3b73f2e2e9632748180ab5b1b Mon Sep 17 00:00:00 2001
From: DiaDust <diadust0514@gmail.com>
Date: Sun, 11 Aug 2013 19:10:16 +0900
Subject: [PATCH] cpufreq : allow cpufreq CA7 100MHz set min clock 200Mhz

---
 drivers/cpufreq/exynos5410-cpufreq.c | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/drivers/cpufreq/exynos5410-cpufreq.c b/drivers/cpufreq/exynos5410-cpufreq.c
index c96f935..997cd66 100644
--- a/drivers/cpufreq/exynos5410-cpufreq.c
+++ b/drivers/cpufreq/exynos5410-cpufreq.c
@@ -825,11 +825,7 @@ static void __init set_volt_table_CA7(void)
 		max_support_idx_CA7 = L7;
 	}
 
-	exynos5410_freq_table_CA7[L9].frequency = CPUFREQ_ENTRY_INVALID;
-	exynos5410_freq_table_CA7[L10].frequency = CPUFREQ_ENTRY_INVALID;
-	exynos5410_freq_table_CA7[L11].frequency = CPUFREQ_ENTRY_INVALID;
-
-	min_support_idx_CA7 = L8;
+	min_support_idx_CA7 = L11;
 }
 
 static void __init set_volt_table_CA15(void)
-- 
1.8.4


